scripts/release.js

```diff
@@ -156,6 +156,19 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install the component app as dependency and build the test app to ensure no errors or build issues
+process.chdir(path.join(__dirname, '..'));
+spawnOrFail('yalc', ['publish']);
+process.chdir(path.join(__dirname, '../..'));
+spawnOrFail('npx', ['create-react-app dependency-check-app']);
+process.chdir(path.join(__dirname, '../../dependency-check-app'));
+spawnOrFail('yalc', ['add amazon-chime-sdk-component-library-react']);
+spawnOrFail('npm', [`install amazon-chime-sdk-js@${updatedSdkVersion}`]);
+spawnOrFail('npm', ['install --save styled-components styled-system']);
```

Comment:
This can be done with the previous install of js sdk.

Reply:
You mean the previous steps (https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/scripts/release.js#L125) already install the js sdk as peer and dev dependency of component library? We don't need to install the js sdk here?

Reply:
No i just mean you can combine the two commands to npm install --save amazon-chime-sdk-js@${updateSdkVersion}  styled-components styled-system

Reply:
Gotcha. Will combine them into one. Thanks